execution_engine/src/core/engine_state/put_trie.rs

```diff
@@ -0,0 +1,33 @@
+use crate::shared::newtypes::Blake2bHash;
+
+/// Result from inserting a [Trie][storage::trie::Trie] into the trie store.  Contains:
+///
+/// - The trie key the inserted [Trie][storage::trie::Trie] (which is its [Blake2bHash])
+///
+/// - The missing descendant keys of that trie transitively refers to. Found using
+///   [storage::global_state::StateProvider::missing_trie_keys].
+#[derive(Debug)]
+pub struct InsertedTrieKeyAndMissingDescendants {
```

Comment:
LMAO

Reply:
That sounds like it's the results from already _inserting_ the missing descendants.